inst/@sym/horzcat.m

```diff
@@ -62,7 +62,7 @@
           'return sp.Matrix.hstack(*_proc),'
           };
 
-  varargin = sym(varargin);
+  varargin = cell_array_to_sym(varargin);
```

Comment:
This change (and other similar ones) are required because after converting `sym` to `classdef`, octave is not allowing to call the constructor with a cell array!!!
That is, `sym({1})` fails!!! I tried with some test classdef classes but the same error didn't appear. Something is different/weird with respect to `sym` only!